proposals/description/swap_dpi_to_dai.ts

```diff
@@ -49,6 +49,33 @@ const swap_dpi_to_dai: ProposalDescription = {
         ]
       ],
       description: 'Remove DPI Deposit wrapper from CR oracle, as now empty'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x471cfe1a44bf1b786db7d7104d51e6728ed7b90a35394ad7cc424adf8ed16816', // SWAP_ADMIN_ROLE
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e' // GOVERN_ROLE
+      ],
+      description: 'Transfer admin of SWAP_ADMIN_ROLE to GOVERNOR, so it can be granted to the TribalCouncil'
```

Comment:
just grant DAO timelock the ROLE_ADMIN.. Shouldn't be changing roles like this

Reply:
yep good point, missed that it already had ROLE_ADMIN so this was redundant